lib/codelog/clis/interactive.rb

```diff
@@ -0,0 +1,54 @@
+require 'thor'
+
+module Codelog
+  module CLIs
+    class Interactive < Thor
+      include Thor::Actions
+
+      no_commands do
+        def initialize
+          @sections = YAML.load_file('changelogs/template.yml').keys
+        end
+
+        def ask_for_changes
+          change_type = ask_for_type
+          say "\nType the entries for \e[33m#{change_type}\e[0m(ENTER to stop):"
```

Comment:
Is it possible to use `Thor`'s `#set_color` method to make the output more legible?
http://www.rubydoc.info/github/wycats/thor/Thor/Shell/Color

Reply:
I think that you are talking about the method parameter and not the output, because it will be the same in either of the two approaches. I don't really think that it will be so much better than you think, but I will change it, so you can approve my changes. 😀

Reply:
I’m talking specifically about the `\e[33m#{change_type}\e[0m` part of the string, it seems a little awkward for me. But I don’t think this change is extremely necessary, it’s just extreme nitpicking. My greatest concern about this PR is the way the recursion is implemented. I think we can elaborate that a little better to declutter the code and make it more legible.